src/Common.ml

```diff
@@ -17,11 +17,11 @@ type flag =
   | Private
       (** An F* private qualifier. *)
   | WipeBody
-      (** Should now be unused. Left there for compatibility with previous ASTs. *)
+      (** DEPRECATED. Left there for compatibility with previous ASTs. *)
```

Comment:
Would it be worth using OCaml's `deprecated` attribute for this constructors (and the ones below)?

Reply:
thanks, I've rebased and fixed this